modules/fedimint-ln-client/src/lib.rs

```diff
@@ -95,8 +105,27 @@ pub trait LightningClientExt {
     ) -> anyhow::Result<UpdateStreamOrOutcome<'_, LnReceiveState>>;
 }
 
-/// The high-level state of a reissue operation started with
-/// [`LightningClientExt::pay_bolt11_invoice`].
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub enum PayType {
+    // Payment from this client to another user within the federation
+    Internal(OperationId),
+    // Payment from this client to another user, facilitated by a gateway
```

Comment:
This enum is also used in the receive case though too right? Not just Outgoing

Reply:
No. `PayType` enum is used only to differentiate the two cases of `pay_bolt11_invoice` action by the fedimint client

Reply:
Ah ok I was confused, thanks!